contracts/StablecoinConverter.sol

```diff
@@ -251,6 +256,7 @@ contract StablecoinConverter is EpochTokenLocker {
             Order memory order = orders[owners[i]][orderIds[i]];
             require(checkOrderValidity(order, batchIndex), "Order is invalid");
             (uint128 executedBuyAmount, uint128 executedSellAmount) = getTradedAmounts(volumes[i], order);
+            require(executedSellAmount > AMOUNT_MINIMUM, "Executed sell amount less than AMOUNT_MINIMUM");
```

Comment:
Would it be more coherent to check `executedBuyAmount > AMOUNT_MINIMUM` here?

Reply:
`executedBuyAmount` is already checked at the top of the function when we `verifyAmounts`. Yet another reason why we should be calling the input parameter `buyVolumes` and not just `volumes`.

Reply:
I agree with the rename. I'd still argue it would be logically more coherent to check them here INSTEAD of above.

Reply:
I could also do that as well.